server/src/test/java/org/elasticsearch/index/mapper/KeywordFieldMapperTests.java

```diff
@@ -773,7 +773,7 @@ public void testDocValuesLoadedFromStoredSynthetic() throws IOException {
     }
 
     public void testFieldTypeWithSkipDocValues_LogsDbMode() throws IOException {
-        assumeTrue("Needs feature flag to be enabled", FieldMapper.DOC_VALUES_SPARSE_INDEX.isEnabled());
+        assumeTrue("Needs feature flag to be enabled", IndexSettings.DOC_VALUES_SKIPPER.isEnabled());
```

Comment:
This change will not be here once we merge https://github.com/elastic/elasticsearch/pull/121751